companion/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-PROJECT( companion )
```

Comment:
It is a left over of the conversion process and can be changed back. It was used as a target name before, now I introduced COMPANION_NAME that is used for target name.

The original problem was that one can not rename a target when installing (`INSTALL( TARGETS `) like it is possible when installing normal files (`INSTALL( FILES`).